### ISSUE TITLE
Missing ctrl+Q keyboard shortcut to quit

### ISSUE THREAD

**Issue by sciunto** [enthought/mayavi#681]
Hi

Thanks @prabhuramachandran for your talk at scipy conference this year.

If I consider your first example

```import numpy as np
x, y = np.mgrid[-3:3:100j, -3:3:100j]
z = np.sin(x**2 + y**2)

from mayavi import mlab
mlab.surf(x, y, z)
```

The problem I'm facing is the impossibility to close the window. Some window managers (such as i3) control the windows only by key strokes. The top bar (reduce, maximize, close) never appears.
Usually, GUIs embed shortcuts like ctrl+Q to quit the apps.

Would it be possible to integrate that to mlab?

Thanks.

**Comment by prabhuramachandran** [comment]
@sciunto -- Try, `mlab.close()`

**Comment by sciunto** [comment]
@prabhuramachandran This doesn't work. I have to use mlab.show() which is hanging. See screenshots.
![s1](https://user-images.githubusercontent.com/335370/42930239-c03ad3d4-8b3c-11e8-9cd6-29456b3008ce.png)
![s2](https://user-images.githubusercontent.com/335370/42930241-c073e0fc-8b3c-11e8-81a3-cbbcdf48b500.png)
![s3](https://user-images.githubusercontent.com/335370/42930243-c08be2b0-8b3c-11e8-9d02-defc1cbfda97.png)

I'm using pyQT5

**Comment by prabhuramachandran** [comment]
Firstly, on a notebook or on an interactive IPython prompt you should first run `%gui qt` so as to let the UI work.  If you do not do this, the UI mainloop will not run.  This is what is done when you do `%matplotlib` for example.  See the docs here http://docs.enthought.com/mayavi/mayavi/mlab.html

Now, as regards closing the window.   It should honor your WM's close window button, so `Alt+F4` or the equivalent should close it. On OSX it does close when you press `Cmd+Q`.

**Comment by sciunto** [comment]
Indeed, `%gui qt` prevents to write mpl.show(), but still, ctrl+Q does not close the window (on Linux with i3) :)
